SwiftWisdom/Core/UIKit/UIApplication+Extensions.swift

```diff
@@ -10,11 +10,10 @@ import UIKit
 
 extension UIApplication {
     public func ip_openSettingsApp() {
-        guard
-            let url = NSURL(string: UIApplicationOpenSettingsURLString)
-            where canOpenURL(url)
-            else { fatalError("Unable to go to settings") }
-        
+        guard let url = URL(string: UIApplicationOpenSettingsURLString), canOpenURL(url) else {
+            // TODO: review - should this be a fatal error?
```

Comment:
I think this should throw actually - could we punt this api decision for the individual review of this file? I'm trying to make as little changes as I can so that we can start a swift 3 branch - this code will not be merged to master.